eclipse_plugin/.classpath

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="src" path="src"/>
+	<classpathentry exported="true" kind="lib" path="lib/guava-19.0.jar"/>
```

Comment:
The parent pom defines 19.0 -- how do I tell Eclipse/Tycho to use the dependecies from there?

And btw. will this PR the base for going open source w/ the Eclipse plug-in? That'd be nice, but I thought, you already have some ready-to-use plug-in ... which just needs to be published?